workflow-templates/binder_badge.properties.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "Add binder when PR is opened/reopened",
+    "description": "Add a badge to launch in mybinder.org."
```

Comment:
`iconName` is required - see https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/sharing-workflows-with-your-organization#creating-a-workflow-template

Reply:
logo added